3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/EnrichmentManager.scala

```diff
@@ -39,6 +39,9 @@ import enrichments.{MiscEnrichments => ME}
 import enrichments.{ClientEnrichments => CE}
 import web.{PageEnrichments => WPE}
 
+// For CurrencyConversion Enrichment
+import com.snowplowanalytics.snowplow.enrich.common.enrichments.registry.TransactionAmounts
```

Comment:
You can start this import at registry:

     import registry.TransactionAmounts